diff --git a/src/freetype/src/gzip/zconf.h b/src/freetype/src/gzip/zconf.h
index 3abf0ba..812766e 100644
--- a/src/freetype/src/gzip/zconf.h
+++ b/src/freetype/src/gzip/zconf.h
@@ -215,9 +215,7 @@
 #   define FAR
 #endif
 
-#if !defined(MACOS) && !defined(TARGET_OS_MAC)
 typedef unsigned char  Byte;  /* 8 bits */
-#endif
 typedef unsigned int   uInt;  /* 16 bits or more */
 typedef unsigned long  uLong; /* 32 bits or more */
 
