diff --git a/OgreMain/src/OgreHlmsDatablock.cpp b/OgreMain/src/OgreHlmsDatablock.cpp
index b11b462..03156d5 100644
--- a/OgreMain/src/OgreHlmsDatablock.cpp
+++ b/OgreMain/src/OgreHlmsDatablock.cpp
@@ -124,7 +124,7 @@ namespace Ogre
         mAlphaTestCmp( CMPF_ALWAYS_PASS ),
         mAlphaTestShadowCasterOnly( false ),
         mAlphaTestThreshold( 0.5f ),
-        mShadowConstantBias( 0.01f )
+        mShadowConstantBias( 0.2f )
     {
         mMacroblockHash[0] = mMacroblockHash[1] = 0;
         mMacroblock[0] = mMacroblock[1] = 0;
